MaxText/train.py

```diff
@@ -340,6 +340,9 @@ def setup_train_loop(config):
   state, state_mesh_annotations, data_iterator = max_utils.setup_training_state(model, data_iterator,
           tx, config, init_rng, mesh, checkpoint_manager)
 
+  max_logging.log('Validating shardings of layers')
+  assert maxtext_utils.is_sharded_correctly(state.params)
```

Comment:
Probably put the assert into the check? But just stylistic.

Reply:
Done. thanks.